### ISSUE TITLE
GBIF no longer displaying images

### ISSUE THREAD

**Issue by sharpphyl** [ArctosDB/arctos#4989]
@dbloom  @dustymc 

**Describe the bug**
In Arctos DMNS:Inv has over 1100 images.  In the past, these were displayed in the GBIF gallery for the collection.  Starting (at least) last week, there are no images attached to DMNS:Inv.

**To Reproduce**
Go to GBIF, select DMNS:Inv and gallery

**Screenshots**
![DMNS clip](https://user-images.githubusercontent.com/15368365/187541027-a8c4fa70-ff5d-4f13-b417-e93d56c5c28d.PNG)

**Data**
Following https://arctos.database.museum/guid/DMNS:Inv:30858 to [GBIF](https://www.gbif.org/occurrence/2147657265), it shows that there is media and links to it in Arctos but has no media identifier.
![clip 2](https://user-images.githubusercontent.com/15368365/187541610-1cea2a8b-0676-4a36-ae8d-9d4ffef10dd3.PNG)

Some DMNS:Bird records have images in GBIF and only a link to Arctos on other images.  See https://www.gbif.org/occurrence/1455538624

![Clip 3](https://user-images.githubusercontent.com/15368365/187542407-874d3a0d-7308-4e32-a17a-7f0450a04c16.PNG)

No media identifier:  http://arctos.database.museum/MediaSearch.cfm?collection_object_id=26494128 (url:https://arctos.database.museum/MediaSearch.cfm?action=search&specimen_id=26494128)

Image in GBIF: http://arctos.database.museum/media/10566373?open (url: https://www.gbif.org/tools/zoom/simple.html?src=//api.gbif.org/v1/image/unsafe/http%3A%2F%2Farctos.database.museum%2Fmedia%2F10566373%3Fopen)

**Desktop (please complete the following information):**
 - Chrome/PC

**Priority**
Not impacting our work but limiting usefulness of records in GBIF.

**Comment by dbloom** [comment]
This is curious @sharpphyl @dustymc.

I've checked the IPT from my end and the tables and links are all working as expected. From my vantage point I can only see a preview of 20 or so records so I can't check to see what might be happening. 

When I viewed your example, however, something odd happened. Perhaps it may give Dusty a clue.

I visited https://www.gbif.org/occurrence/1455538624 to look at the image for which a preview is not shown in the above comment, but when I click on the References link for the missing image - http://arctos.database.museum/MediaSearch.cfm?collection_object_id=26494128 - it takes me to a page that shows only 4 images, not 5 as implied in the occurrence.
![Screen Shot 2022-08-30 at 2 27 10 PM](https://user-images.githubusercontent.com/942447/187546635-a6a139e5-e477-4c4c-9470-4e2d522bf6e5.png)

Is it possible that there is an extra, but incomplete line in the Arctos table? Based on this alone, it would suggest that there should only be 4 images. If there should be 5 then this might be a back end issue on the Arctos side.
